src/renderer/state/setting/reducer.ts

```diff
@@ -28,6 +28,7 @@ export const initialState: SettingStateBranch = {
     [AlertType.INFO]: true,
     [AlertType.DRAFT_SAVED]: false,
   },
+  shouldStoreLocally: true,
```

Comment:
I think this state field makes more sense in the `selection` state branch IMO
https://github.com/aics-int/aics-file-upload-app/blob/a48c70336d0e3d4bb7c8146854e43338482846b9/src/renderer/state/selection/reducer.ts

Reply:
resolved in [4e933f6](https://github.com/aics-int/aics-file-upload-app/pull/116/commits/4e933f6acba2f622da8d9fe9a79869cc552ee5a0)

Reply:
I think thats a safer way to do it.